PhpAmqpLib/Connection/AbstractConnection.php

```diff
@@ -559,26 +559,25 @@ protected function wait_channel($channel_id, $timeout = 0)
                 }
                 continue;
 
-            } else {
```

Comment:
No, the `else` is there in case `if ($frame_channel === 0 && $frame_type === 8)` doesn't match. The `else` and the `continue` are totally unrelated